README.txt

```diff
@@ -706,6 +706,260 @@
+  	--CanvMarginTop=CANVMARGINTOP
+                        Set Canvas Margin Top (default is 0.08)
+  	--CanvMarginBot=CANVMARGINBOT
+                        Set Canvas Margin Bottom (default is 0.14)
+  	--CanvMarginLf=CANVMARGINLF
+                        Set Canvas Margin Left (default is 0.16)
+  	--CanvMarginRt=CANVMARGINRT
+                        Set Canvas Margin Right (default is 0.06)
+  	--CanvPlotType=CANVPLOTTYPE
+                        Set the Canvas Plot type (default is TGraphErrors)
+  	--CanvTitleX=CANVTITLEX
+                        Set Xaxis title, example: Divider Current
+                        #left(#muA#right)
+  	--CanvTitleY=CANVTITLEY
+                        Set Yaxis title, example: Applied Voltage
+                        #left(kV#right)
+  	--CanvRangeX=CANVRANGEX
+                        Set the X-Range of the Canvas (default is 0,1000)
+  	--CanvRangeY=CANVRANGEY
+                        Set the Y-Range of the Canvas (default is 0,7)
+  	--CanvTitleOffsetX=CANVTITLEOFFSETX
+                        Set the X-Offset of the Canvas Title (default is 1.0)
+  	--CanvTitleOffsetY=CANVTITLEOFFSETY
+                        Set the Y-offset of the Canvas Title (default is 0.8)
+  	--CanvName=CANVNAME   Set Canvas Name, Default is: LS2_Detectors
+  	--YaxisScale          If YaxisScale option is used the Y axis is plotted in
+                        kUnit. Default is: False
+  	--SetErrX             Set Error option true (for X axis)
+  	--SetErrY             Set Error option true (for Y axis)
+  	--SelectColumnErrX=SELECTCOLUMNERRX
+                        If SetErrX=True choose the Column for XError, Default
+                        is: 0
+  	--SelectColumnErrY=SELECTCOLUMNERRY
+                        If SetErrY=True choose the Column for YError, Default
+                        is: 0
+  	--PlotLineSize=PLOTLINESIZE
+                        Set Plot Line Size (default is: 1)
+  	--PlotLineStyle=PLOTLINESTYLE
+                        Set Plot Line Style (default is: 1)
+  	--PlotMarkerSize=PLOTMARKERSIZE
+                        Set Plot Marker Size (default is: 0.8)
+  	--Fit                 if it is used the header parameters for the fit are
+                        created
+  	--FitFormula=FITFORMULA
+                        Set the fit formula e.g. [0]*x^2+[1]
+  	--FitLineSize=FITLINESIZE
+                        Set Fit Line Size, default is: 1
+  	--FitLineStyle=FITLINESTYLE
+                        Set Fit Line Style, default is: 1
+  	--FitOption=FITOPTION
+                        Set Fit Option, default is: R
+  	--FitParamIGuess=FITPARAMIGUESS
+                        Set Fit Parameters initial values
+  	--FitPerform=FITPERFORM
+                        if true (default) perform a fit to the TObject defined
+                        in the [BEGIN_PLOT]header this [BEGIN_FIT] header is
+                        found in
+  	--FitRange=FITRANGE   Set the Fit range, default is 0,1
```

Comment:
You need to merge your branch with the central branch.  Your branch has diverged and does not include the changes.

Additionally re-write this section in [MarkDown](https://guides.github.com/features/mastering-markdown/).  Don't just dump the results of `Produce_Config_File.py -h` to the terminal.  Make them in a table as shown [here](https://github.com/bdorney/CMS_GEM_Analysis_Framework/blob/develop/README.md).